components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,464 @@
+
+momentum:
+
+$$
+\frac{D \rho {\bf u} }{D t} \equiv
+\frac{\partial \rho {\bf u}}{\partial t}
+ + \nabla \cdot \left( \rho {\bf u} \otimes {\bf u}  \right)
++ w\frac{\partial \rho {\bf u}}{\partial z}
+  = - \nabla p 
+- \rho f {\bf k} \times {\bf u}
++ \rho {\bf D}^u + \rho {\bf F}^u 
+$$ (continuous-momentum-R13App)
+
+mass:
+
+$$
+\frac{D \rho}{D t} \equiv
+\frac{\partial \rho }{\partial t}
+ + \nabla \cdot \left( \rho  {\bf u}  \right)
+ + \frac{\partial}{\partial z} \left(  \rho  w \right)
+= 0
+$$ (continuous-mass-R13App)
+
+tracer:
+
+$$
+\frac{D \rho \varphi }{D t} \equiv
+\frac{\partial \rho \varphi}{\partial t}
+ + \nabla \cdot \left( \rho \varphi {\bf u}  \right)
+ + \frac{\partial}{\partial z} \left(  \rho \varphi w \right)
+= D^\varphi + S^\varphi
+$$ (continuous-tracer-R13App)
+
+Here we have express the following terms as a general operators, with examples of specific forms provided below: the dissipation ${\bf D}^u$, momemtum forcing ${\bf F}^u$, tracer diffusion $D^\varphi$, and tracer sources and sinks $S^\varphi$. The momentum equation can be rewritten using the product rule on $\rho {\bf u}$, mass conservation, and dividing by $\rho$, as:
+
+$$
+\frac{D {\bf u} }{D t} \equiv
+\frac{\partial {\bf u}}{\partial t}
+ + \nabla \cdot \left( {\bf u} \otimes {\bf u}  \right)
++ w\frac{\partial {\bf u}}{\partial z}
+  = - \frac{1}{\rho} \nabla p 
+- f {\bf k} \times {\bf u}
++ {\bf D}^u + {\bf F}^u
+$$ (continuous-momentum-rho)
+
+Note that the advective term may be replaced with the vector identity
+
+$$
+\nabla \cdot \left( {\bf u} \otimes {\bf u}  \right) = 
+\boldsymbol{u} \cdot \nabla \boldsymbol{u} &= (\nabla \times \boldsymbol{u}) \times \boldsymbol{u} + \nabla \frac{|\boldsymbol{u}|^2}{2} \\
+&= \{\boldsymbol{k} \cdot (\nabla \times \boldsymbol{u})\} \boldsymbol{k} \times \boldsymbol{u} + \nabla \frac{|\boldsymbol{u}|^2}{2} \\
+&= \zeta \boldsymbol{u}^{\perp} + \nabla K.
+$$ (advection-identity)
+
+where $\zeta$ is relative vorticity and $K$ is kinetic energy. This is used to produce the final formulation below.
+
+For a primitive equation ocean model, we assume the fluid is incompressible and hydrostatic. For Omega we are not making the Boussinesq assumption, so all density terms are the full density. In particular, the density coefficient of the pressure gradient in [](#continuous-momentum-R13App) is not a constant, as it is in primitive equation models like POP and MPAS-Ocean.
+
+**Incompressible:** Assume that the three-dimensional velocity field is nondivergent,
+
+$$
+ \nabla \cdot {\bf u} + \frac{\partial w}{\partial z}  = 0.
+$$ (continuous-incompressible)
+
+**Hydrostatic:** Beginning with the vertical momentum equation,
+
+$$
+\frac{D w }{D t} 
+  = 
+  - \frac{1}{\rho} \frac{\partial p}{\partial z} - g + {\bf D}^u + {\bf F}^u
+$$ (continuous-vert-mom)
+
+assume that advection of vertical momentum $Dw/Dt$, dissipation, and forcing are small, and that the first order balance is between pressure gradient and gravity,
```

Comment:
i feel this is more appropriate to say the balance is between pressure and buoyancy.

Reply:
done.